signwriting/visualizer/test_visualize.py

```diff
@@ -55,10 +54,9 @@ def test_image_small_box_is_corrected(self):
         assert_array_equal(np.array(image), np.array(reference_image))
 
     def test_image_with_line_and_embedded_color(self):
-        fsw = "M518x518S30a00482x483S33e00482x483"
-
-        image = signwriting_to_image("M518x518S30a00482x483S33e00482x483",
-                                     line_color=(144, 70, 180, 255), embedded_color=True)
+        fsw = "M530x518S19a30500x482S19a38465x481S22f04509x506S22f14467x504"
+        image = signwriting_to_image(fsw, line_color=(
+            10, 23, 122, 255), fill_color=(255, 255, 255, 0))
```

Comment:
i don't like where the line breaks - either new line before `line_color` or same line with the color

Reply:
Stupid formatter